### PR TITLE
EARTH-86

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Various homepage specific and paragraph tweaks
- Add body classes for front page and not front pages

# Needed By (Date)
- N/A

# Urgency
- Not critical

# Steps to Test

1. Pull this branch
2. Ensure that classes are dynamically being added to the <body>
3. See various style tweaks

# Affected Projects or Products
- Earth
- Matson

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/EARTH-86

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by sherakama — APPROVED** [review]

File: scss/components/filmstrip/_filmstrip.scss [hunk 1]
```diff
@@ -11,6 +11,12 @@
 
 // Stanford Earth styling for the filmstrip top-border
 .filmstrip {
+  .front & {
```
Comment (sherakama) [inline scss/components/filmstrip/_filmstrip.scss]: It is very possible that the front page have more than one filmstrip. Infact, I know it does. Please revise this to a front page specific stylesheet that targets the specific block you are trying to pull up over another element. This is far too general.

File: scss/components/highlight-banner/_highlight-banner.scss [hunk 2]
```diff
@@ -9,15 +9,27 @@
...
-  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
-  padding: 30px 30px 20px;
-
-  p {
-    color: color(ash);
-    letter-spacing: .5px;
-    line-height: 2em;
-    margin-bottom: .5em;
+.highlight-banner__container {
+  .front & {
```
Comment (sherakama) [inline scss/components/highlight-banner/_highlight-banner.scss]: Same as above. This should be pulled out in to a home page specific stylesheet.

File: scss/components/highlight-banner/_highlight-banner.scss [hunk 3]
```diff
@@ -9,15 +9,27 @@
...
+    top: -35px;
+  }
+
+  .highlight-cards {
+    .highlight-card {
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+      padding: 30px 30px 20px;
+
+      .front & {
```
Comment (sherakama) [inline scss/components/highlight-banner/_highlight-banner.scss]: Are you sure you want all highlight cards on the home page for all of time to have a margin bottom of this? What if others get added or moved over time?

File: templates/html.html.twig [hunk 4]
```diff
@@ -48,13 +48,11 @@
     <css-placeholder token="{{ placeholder_token }}">
     <js-placeholder token="{{ placeholder_token }}">
   </head>
-  {% set classes = [] %}
+  {% set classes = [is_front ? 'front', not is_front ? 'not-front'] %}
```
Comment (sherakama) [inline templates/html.html.twig]: nice.

**Review by sherakama — CHANGES_REQUESTED** [review]
@josephgknox 
Please have a review of the comments. 

Thanks,

File: templates/html.html.twig (hunk 4 above)
Comment (pookmish) [reply templates/html.html.twig]: couldn't this be `is_front ? 'front' : 'not-front'`

Comment (josephgknox) [reply templates/html.html.twig]: Thanks @pookmish. Good catch, will keep this cleaner.

**Comment by josephgknox** [comment]
@sherakama ready for yah. :o)

**Review by sherakama — APPROVED** [review]

**Pull request merged**